tests/deepsparse/pipelines/test_dynamic_import.py

```diff
@@ -15,7 +15,8 @@
 import os
 
 import pytest
-from deepsparse.pipeline import _REGISTERED_PIPELINES, Pipeline
+from deepsparse.base_pipeline import _REGISTERED_PIPELINES
```

Comment:
ditto for `_REGISTERED_PIPELINES`

Reply:
the `register` function is now part of the `BasePipeline` so I'm not sure if it makes sense for `_REGISTERED_PIPELINES` to live in `pipline.py`?